package.json

```diff
@@ -18,9 +18,15 @@
     "url": "https://github.com/nervnews/nerv/issues"
   },
   "homepage": "https://github.com/nervnews/nerv#readme",
+  "devDependencies": {
+    "morgan": "^1.9.0",
+    "nodemon": "^1.14.11"
+  },
   "dependencies": {
     "body-parser": "^1.18.2",
+    "emotional": "0.0.3",
     "express": "^4.16.2",
-    "nodemon": "^1.14.11"
+    "polarity": "^3.0.1",
+    "sentiment": "^4.1.0"
```

Comment:
whats this?

Reply:
module for sentiment analysis!

Reply:
https://github.com/nervnews/nerv/wiki/Sentiment-Analysis